bundles/dev/uds-bundle.yaml

```diff
@@ -156,7 +156,7 @@ packages:
               path: "gitlab.webservice.resources"
               default:
                 limits:
-                  memory: 2.5G
+                  memory: 3.5G
```

Comment:
This all looks fine, just requesting changes to block until i've validated testing locally.